os/audio/audio.c

```diff
@@ -693,6 +693,42 @@ static inline void audio_dequeuebuffer(FAR struct audio_upperhalf_s *upper, FAR
 	}
 }
 
+/****************************************************************************
+ * Name: audio_handleerror
+ *
+ * Description:
+ *   Send an AUDIO_MSG_IOERR message to the client to indicate that the
+ *   an error has occured.  The lower-half driver initiates this
+ *   call via its callback pointer to our upper-half driver.
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_AUDIO_MULTI_SESSION
+static inline void audio_handleerror(FAR struct audio_upperhalf_s *upper, FAR struct ap_buffer_s *apb, uint16_t status, FAR void *session)
+#else
+static inline void audio_handleerror(FAR struct audio_upperhalf_s *upper, FAR struct ap_buffer_s *apb, uint16_t status)
+#endif
+{
+	struct audio_msg_s msg;
+
+	audvdbg("Entry\n");
```

Comment:
It is useless here.

Reply:
Done

Reply:
please check this one more time, log is still here.

Reply:
Sorry. I missed in my last commit. 
Fixed now.